contracts/PoolManager.sol

```diff
@@ -24,15 +24,13 @@ contract PoolManager is IPoolManager, NoDelegateCall, ERC1155, IERC1155Receiver
     /// @inheritdoc IPoolManager
     int24 public constant override MAX_TICK_SPACING = type(int16).max;
 
+    /// @inheritdoc IPoolManager
+    int24 public constant override MIN_TICK_SPACING = 1;
```

Comment:
I don't think this needs to be public. Removing methods like this can save gas on all operations because solidity often structures large switches as sequential if-statements.

Reply:
https://github.com/Uniswap/v4-core/pull/810